sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/PartitionPumpManager.java

```diff
@@ -193,10 +195,9 @@ private Context startProcessTracingSpan(EventData eventData, String eventHubName
             return Context.NONE;
         }
 
-        Context spanContext = tracerProvider.extractContext(diagnosticId.toString(), Context.NONE);
-        Context entityContext = spanContext.addData(ENTITY_PATH_KEY, eventHubName);
-
-        return tracerProvider.startSpan(entityContext.addData(HOST_NAME_KEY, fullyQualifiedNamespace),
+        Context spanContext = tracerProvider.extractContext(diagnosticId.toString(), Context.NONE)
+            .addData(ENTITY_PATH_KEY, eventHubName).addData(HOST_NAME_KEY, fullyQualifiedNamespace);
+        return tracerProvider.startSpan(spanContext.addData(AZ_NAMESPACE_KEY, AZ_NAMESPACE_VALUE),
```

Comment:
it would be easier to reason about if you chained `.addData(NAMESPACE...)` to the declaration above rather than here.